mappings/net/minecraft/world/level/redstone/NeighborUpdater.mapping

```diff
@@ -0,0 +1,25 @@
+CLASS net/minecraft/unmapped/C_rwmwglbe net/minecraft/world/level/redstone/NeighborUpdater
+	FIELD f_ccnhxzee DIRECTIONS [Lnet/minecraft/unmapped/C_xpuuihxf;
+	METHOD m_cmnwwuvp executeShapeUpdate (Lnet/minecraft/unmapped/C_vdvbsyle;Lnet/minecraft/unmapped/C_xpuuihxf;Lnet/minecraft/unmapped/C_txtbiemp;Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_hynzadkk;II)V
+		ARG 0 world
+		ARG 3 pos1
+		ARG 4 pos2
```

Comment:
better names for these parameters across this whole class would be ideal
"pos1" and "pos2" doesn't tell you what they do